### PR TITLE
Cli fix

### LINKED ISSUE #3: Update cli.py
## Summary

The code was heavily refactored to incorporate jinja templating, but `cli.py` was not updated. 

## Proposed implementation

Walk through `cli.py` and update the code to work with the new codebase.

## Acceptance Criteria

- [ ] `cli.py` runs without errors
- [ ] Tests for `cli.py` pass

## Tasks to complete

- [ ] Update `cli.py` as needed
- [ ] Write tests for `cli.py`

### PR DESCRIPTION
**Pull request by kcpevey**
Resolves #3

### CODE REVIEW

**Comment by iameskild** [comment]
@kcpevey @rycrow I believe the CLI is now ready to go. I made a few minor modifications to the `download_packages` function, namely adding an option to specify the `download_path`. And aside from a tiny fix to `update_repo`, I'm not sure what additional changes (if any) are needed.

File: ib_manifest_util/cli.py [hunk 1]
```diff
@@ -31,31 +16,79 @@ def main() -> None:
...
+    output_hardening_path: str | Path | None = None,
+    output_dockerfile_path: str | Path | None = None,
+):
+    update_repo(
+        repo_dir,
+        dockerfile_version,
+        local_env_path,
+        startup_scripts_path,
+        output_hardening_path,
+        output_dockerfile_path,
```
Comment (kcpevey) [inline ib_manifest_util/cli.py:60]: I added an additional parameter here, `output_dockerfile_template`. Its just been merged into dev. Can you update the cli to accept it?

Merging dev into your branch will probably also fix all those merge conflicts.

**Review by kcpevey — COMMENTED** [review]
LGTM. I just had a request to update based on recent changes in dev.

**Comment by iameskild** [comment]
@kcpevey I've fixed all the merge conflicts and I believe is ready for a final review :)

**Comment by kcpevey** [comment]
This looks great. Thanks @iameskild, this is ready to merge!

**Review by iameskild — APPROVED** [review]

**Pull request merged**